e2e-tests/components/toolbar/components/copy-paste/utils.js

```
@@ -27,50 +27,50 @@ export const block = {
+		'container-max-height-unit-xl': 'px',
+		'container-height-unit-xl': 'px',
+		'container-min-height-unit-xl': 'px',
+		'container-max-width-xl': 1170,
 		'container-max-width-xxl': 1790,
 		'container-max-width-xl': 1170,
 		'container-max-width-l': 90,
 		'container-max-width-m': 90,
 		'container-max-width-s': 90,
 		'container-max-width-xs': 90,
-		'container-max-width-unit-general': 'px',
+		'container-max-width-unit-xl': 'px',
```

Biome (1.9.4)

[error] This property value named container-max-width-unit-xl is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named container-max-width-unit-xl


(lint/suspicious/noDuplicateObjectKeys)